templates/invite_body.html

```diff
@@ -450,24 +454,39 @@
                                       <div class="postNameAndTime">
-                                        <div class="senderName">{{.Props.SenderName}}</div>
-                                        {{if .Props.Time}}
-                                        <div class="time">{{.Props.Time}}</div>
+                                        <div class="senderName">{{.SenderName}}</div>
+                                        {{if .Time}}
+                                        <div class="time">{{.Time}}</div>
                                         {{end}}
-                                        {{if .Props.ChannelName}}
+                                        {{if .ChannelName}}
                                         <div class="channelBg">
                                           <div class="channelLogo"><img src="{{$.Props.SiteURL}}/static/images/channel_icon.png" width=10px height=10px></img></div>
-                                          <div class="channelName">{{.Props.ChannelName}}</div>
+                                          <div class="channelName">{{.ChannelName}}</div>
                                         </div>
                                         {{end}}
                                       </div>
                                     </td>
                                   </tr>
                                   <tr>
                                     <td align="center" class="senderMessage" style="font-size:0px;padding:0px;word-break:break-word;">
-                                      <div style="font-family:Open Sans, sans-serif;font-size:13px;line-height:1;text-align:center;color:#000000;">{{.Props.Message}}</div>
+                                      <div style="font-family:Open Sans, sans-serif;font-size:13px;line-height:1;text-align:center;color:#000000;">{{.Message}}</div>
+                                    </td>
+                                  </tr>
+                                  {{if .MessageURL}}
+                                  <tr>
+                                    <td align="center" vertical-align="middle" class="messageButton" style="font-size:0px;padding:16px 0px 0px 0px;word-break:break-word;">
+                                      <table border="0" cellpadding="0" cellspacing="0" role="presentation" style="border-collapse:separate;line-height:100%;">
+                                        <tr>
+                                          <td align="center" bgcolor="#FFFFFF" role="presentation" style="border:none;border-radius:4px;cursor:auto;mso-padding-alt:10px 25px;background:#FFFFFF;" valign="middle">
+                                            <a href="{{.MessageURL}}" style="display:inline-block;background:#FFFFFF;color:#ffffff;font-family:Open Sans, sans-serif;font-size:13px;font-weight:normal;line-height:120%;margin:0;text-decoration:none;text-transform:none;padding:10px 25px;mso-padding-alt:0px;border-radius:4px;" target="_blank">
```

Comment:
is this template auto-generated? wondering if there is no other way to set this styles in a way that is easier to read and thus to maintain.

Reply:
@pablovelezvidal  yes, this is autogenerated